views/login/code.pug

```diff
@@ -1,23 +1,23 @@
 extends ../base
 
 block variables
-  -var title = __('Enter your personal access code')
+  -var title = __('Enter your personal filing code')
 
 block content
 
   h1 #{title}
 
   div
-    p #{__('Your personal access code is the the 9-character code at the bottom of your notification letter.')}
+    <p>#{__('You can find your personal filing code in the CRA letter about this new')} <i>#{__('Claim Tax Benefits')} </i>#{__('service.')}</p>
```

Comment:
Hi Nick, I have a couple of small comments.

1. Since we're using pug, let's just do `p` and not `<p>`
2. I was going to say to use the `em` tag, but [this post](https://medium.com/@zac_heisey/when-to-use-strong-b-em-and-i-tags-in-your-markup-fa4d0af8affb) says that `i` is right, so carry on.
3. Can you say `'service.name'` instead of `'Claim Tax Benefits'`. Sorry about the inconsistency, but it's the only one like that.